versioned_docs/version-1.3/components/zeebe/zeebe-overview.md

```diff
@@ -15,7 +15,7 @@ With Zeebe you can:
 - Use as part of a software as a service (SaaS) offering with Camunda Cloud or deploy with Docker and Kubernetes (in the cloud or on-premises) with Camunda Cloud Self-Managed.
 - Scale horizontally to handle very high throughput.
 - Rely on fault tolerance and high availability for your processes.
-- Export processes data for monitoring and analysis (currently only available through the [Elasticsearch exporter](https://github.com/camunda-cloud/zeebe/tree/develop/exporters/elasticsearch-exporter) added in Camunda Cloud Self-Managed).
+- Export processes data for monitoring and analysis (currently only available through the [Elasticsearch exporter](https://github.com/camunda-cloud/zeebe/tree/main/zeebe/exporters/elasticsearch-exporter) added in Camunda Cloud Self-Managed).
```

Comment:
I'm not sure what version is this `1.3` so I pointed it to the github `main` README.md